### PR TITLE
Typo fix, colorized output

### PR DESCRIPTION
**Pull request by hzlmn**
What was done:
- Fixed minor typo for `use_redis` prop
- Colorized a bit output for better visual recognition
- Dropped redundant comment in template
colors

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py [hunk 1]
```diff
@@ -12,7 +12,6 @@ def init_routes(app: web.Application) -> None:
 
     add_route('*', '/', index, name='index')
 
-    # added static dir
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: Why did u remove this line?

Comment (hzlmn) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: What is the purpose of this comment? It is pretty clear from code that we are adding static dir. But i can revert no prob.

Comment (Arfey) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: Please return if you can

Comment (hzlmn) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: done

Comment (hzlmn) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: but i still doubt about this comment :)

**Comment by Arfey** [comment]
Thank. The console output looks much better 😄

File: create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py (hunk 1 above)
Comment (webknjaz) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: @hzlmn removing a bunch of unrelated changes is a smelly practice. Avoid doing this.

Comment (hzlmn) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py]: @webknjaz Thanks, i will take note.

**Pull request merged**